packages/lesswrong/lib/analyticsEvents/ultraFeedEvents.ts

```diff
@@ -0,0 +1,130 @@
+import { useRef, useEffect, useCallback } from 'react';
```

Comment:
Any reason why this file is in `lib/analyticsEvents` if it's a React hook?

Reply:
deleted anyway